opentracing-jaxrs2-itest/common/src/main/java/io/opentracing/contrib/jaxrs2/itest/common/AbstractServerTest.java

```diff
@@ -118,6 +118,20 @@ public void testNotExistingURL() throws Exception {
     }
 
     @Test
+    public void testExceptionInHandler() throws Exception {
+        Client client = ClientBuilder.newClient();
+        Response response = client.target(url("/exception"))
+            .request()
+            .get();
+        response.close();
+        Assert.assertEquals(500, response.getStatus());
+
+        List<MockSpan> mockSpans = mockTracer.finishedSpans();
+        // TODO jax-rs interceptors do not trace this
```

Comment:
Would it be worth referencing github issue here that will track adding this?

Reply:
+1